### DISCUSSION TITLE
Created for John. Where are the custom start menu icon code resides?

### DISCUSSION THREAD

**Discussion by coddec** [Open-Shell/Open-Shell-Menu#1133] [Q&A]
Hi There! I was wondering if I could get some information on the Open-Shell Menu project. Specifically, I'm looking for where the custom start menu icon code resides. That is all, and thanks in advance! Let me know if you need any more information.

**Comment by CTVCAM8** [comment]
As I'm not a coder, all I can do is help steer you to two possible locations:

Open-Shell-Menu-master\Src\StartMenu\StartMenuDLL

StartButton.cpp
StartButton.h
